common/dynamicconfig/constants.go

```diff
@@ -361,6 +361,11 @@ const (
 	QueuePendingTaskMaxCount = "history.queuePendingTasksMaxCount"
 	// QueueMaxReaderCount is the max number of readers in one multi-cursor queue
 	QueueMaxReaderCount = "history.queueMaxReaderCount"
+	// EnableCloseWorkflowCleanup to clean up the mutable state after visibility close
+	// task has been processed. Must use Elasticsearch as visibility store, otherwise
+	// workflow data (eg: search attributes) will be lost after workflow is closed.
+	EnableCloseWorkflowCleanup = "history.enableCloseWorkflowCleanup"
```

Comment:
nit: plz group this with other visibility processor/task configs and use the same order in history config.go.

maybe also rename to `cleanupVisibility`? At first glance at the name I have no idea what it's going to clean up.